src/credit-card/credit-card.ts

```diff
@@ -51,5 +51,6 @@ export function encrypt(cardNumber: string|number, cvv: string|number, month: st
   if(!validate(cardNumber, cvv, month, year)){
     return Observable.throw('Credit card details invalid');
   }
-  return creditCardEncrypt(cleanInput(cardNumber), cleanInput(cvv), Number(cleanInput(month)), Number(cleanInput(year)));
+  // allow CVV to be optional if it is null
+  return creditCardEncrypt(cleanInput(cardNumber), cvv === null ? '' : cleanInput(cvv), Number(cleanInput(month)), Number(cleanInput(year)));
```

Comment:
Sending a `''` or `null` in the tokenization request results in TSYS tokenizing successfully and returning `''`. Lee if this implementation is good, it should continue to allow the ERT to require a CVV. Just some of the other apps seem to need it to be optional.